projects/packages/videopress/src/client/lib/url/index.ts

```diff
@@ -129,3 +129,44 @@ export function buildVideoPressURL(
 export const removeFileNameExtension = ( name: string ) => {
 	return name.replace( /\.[^/.]+$/, '' );
 };
+
+/**
+ * Helper function to create and return textarea element.
+ *
+ * > Moreover, using textContent can prevent XSS attacks.
+ * https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#differences_from_innerhtml
```

Comment:
Perhaps we can add a `@see` in front of this link?

Reply:
e1f181ef45aa94238d4c7f2cd53ba49efad9cbf6